### DISCUSSION TITLE
Auto refresh issue?

### DISCUSSION THREAD

**Discussion by jan-Sanku** [vrcx-team/VRCX#588] [General]
For a few days now, all updates in my feed (GPS, online/offline, status, etc) are massively delayed, usually by 10+ minutes, but I've seen it delay up to an *hour*. At the moment, the only temporary fix I could find is hitting the "Refresh Friends" button every time I want to check VRCX, which results in 20+ feed entries catching back up, on average. Just in the wrong order, of course, since they arrive at the same time. Also, things go missing. For example, if a user switches worlds or statuses multiple times while VRCX isn't following, hitting refresh will only show the latest one (this is to be expected given the circumstances, but still undesirable).

This hasn't been an issue before. I always got my notifications and feed entries *right* on time, much faster than I'd see them happen in game. Hitting refresh never did anything, since everything was always up to date.

So what's happening here? Is anyone else experiencing this too?

**Comment by Natsumi-sama** [comment]
Can you try latest VRCX nightly build?

**Reply by jan-Sanku** [reply]
Sure thing, updating now. It'll take a while to tell if this still happens, of course, but I'll get back to you.

**Reply by jan-Sanku** [reply]
So far it's looking pretty good! Didn't get to try notifications yet (so no real time reference), but manually refreshing in VRCX doesn't seem to do anything anymore, and that's a good sign. Pretty sure this fixed it!

**Comment by jan-Sanku** [comment]
Also, this started happening around the same time, might be related. If not, I'll make a separate thread.

Had to censor the names, but I marked which ones go together. My VRCX has been thinking people are somehow changing ranks all the time. Hasn't done it before, and started around the same time as the refresh issues. First I thought maybe VRC now reflects hiding one's rank in the API, but you can't hide as "known" if you're "trusted".

It seems very odd. Let me know if you think this is related.

![trustlevel](https://github.com/vrcx-team/VRCX/assets/56739341/d3e186fc-a2ec-46ff-a3da-ff97d5b2c0f7)

**Comment by Natsumi-sama** [comment]
VRChat API has seemingly undergone many changes these part few days causing all sorts of weird behavior.

**Reply by jan-Sanku** [reply]
Ah I see, good to know. Hope they resolve that soon.
